libraries/AP_Proximity/AP_Proximity_LightWareSF40C_v09.cpp

```diff
@@ -205,7 +202,7 @@ bool AP_Proximity_LightWareSF40C_v09::send_request_for_distance()
     char request_str[16];
     snprintf(request_str, sizeof(request_str), "?TS,%u,%u\r\n",
              (unsigned int)PROXIMITY_SECTOR_WIDTH_DEG,
-             _sector_middle_deg[_last_sector]);
+             boundary._sector_middle_deg[_last_sector]);
```

Comment:
I'm very tempted to remove support for the Lightware SF40c_v09 because its interface is so different to other sensors that maintenance is difficult.  It was muramura who convinced me to keep supporting it but I suspect he and I are perhaps the almost the only AP users that still have these sensors around.

Reply:
To be more specific, this is the only driver that requires knowledge of the size of sectors and that directly injects the distances into _sector_middle_deg.